apps/pie-docs/src/foundations/typography/overview.md

```diff
@@ -136,18 +137,18 @@ We use this style when indicating something is no longer valid, but can still be
 ---
 
 ## Alignment
-This property defines the horizontal alignment of the text. It includes three types of aligment:
+This property defines the horizontal alignment of the text. It includes three types of alignment.
 
 ### Left aligned
-This is the preferred way to align text within our products. Left-aligned text performs better for readers because it helps the users to easily identify the start of a new line.
+This is the preferred way to align text within our products. Left-aligned text performs better for readers because it helps users to easily identify the start of a new line.
```

Comment:
<img src="https://user-images.githubusercontent.com/26894168/226684543-306a3ce7-e85a-4dae-bc1e-d6c52581ea70.png" width="300px" />


```suggestion
This is the preferred way to align text within our products because it makes it easier to identify the start of a new line (for left-to-right languages).
```